Workbooks/CosmosDb/Cost Optimization/Optimizing Total Provisioned Throughput.workbook

```diff
@@ -0,0 +1,1002 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 12,
+      "content": {
+        "version": "NotebookGroup/1.0",
+        "groupType": "editable",
+        "items": [
+          {
+            "type": 1,
+            "content": {
```

Comment:
Also just for easier reading, can we add some color to each of the headings i.e. "Introduction", "Background Information", "Task 1: Analyze your utilization" could be a blue color.

Reply:
nit- can you use the MS blue? 0078D4 or rgb(0,120,212)

Reply:
Can we add some buttons to trigger to go to the next section?

Reply:
I'm not able to see the raw values: 
<img width="756" alt="image" src="https://user-images.githubusercontent.com/29219340/201784670-ca77f0d6-cb06-4117-b2e5-387fe27eb12c.png">